content/md/support/manage-support-access.md

```diff
@@ -70,3 +70,9 @@ Please wait a moment for the change to be effective. The status will first be in
 1. On the `Activity Log` sub-tab, you can see the history of all the access changes.
 
 ![Activity log subtab](../img/helpdesk_activity_log.png)
+
+# Access JIRA SDS Helpdesk
```

Comment:
Please integrate the content as part of "Add an access" section.

Add as warning section with the sentence: Access to the Helpdesk is granted per user by creating a separate account (the authentication is not shared with the Portal).